python/jupytercad_core/jupytercad_core/constants.py

```diff
@@ -0,0 +1 @@
+CURRENT_SCHEMA_VERSION = "3.0.0"
```

Comment:
ideally, we could generate files containing this value from the value in the `jcad.json` file, this will ensure a single source of trust for the schema version. But I'm fine defining them manually.

Reply:
Indeed, I can access that value from 

```python
from jupytercad_lab.notebook.objects._schema.jcad import IJCadContent
IJCadContent.model_fields['schemaVersion'].default
```

If we take that route though it would be cleaner to have the generated Python classes be part of `jupytercad_core`, any objection doing this?

Reply:
When it comes to the TypeScript generated interfaces though, it does not have the schema value information that comes from `jcad.json`. Not sure how to workaround that in that case.

Reply:
Indeed it's cleaner.

Reply:
there is a node function in `packages/schema/schema.json` which reads the json files, you can use it to write your constant somewhere

Reply:
Sounds good

Reply:
I've pushed a commit doing all this :)